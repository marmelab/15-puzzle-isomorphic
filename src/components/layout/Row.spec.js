import React from 'react';
import renderer from 'react-test-renderer';

import Row from './Row';

describe('Row', () => {
    test('should render Row without error', () => {
        renderer.create(
            <Row>
                <div>Child</div>
            </Row>,
        );
    });

    test('should render the Row correctly', () => {
        const component = renderer.create(
            <Row>
                <div>Child</div>
            </Row>,
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
