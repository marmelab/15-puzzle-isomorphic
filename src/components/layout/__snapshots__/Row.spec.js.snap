// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row should render the Row correctly 1`] = `
<div
  className="row"
>
  <div
    className="col s12"
  >
    <div>
      Child
    </div>
  </div>
</div>
`;
